bands/templates/bands/index.html

```diff
@@ -160,6 +173,9 @@
             <br/></br>
         </td>
     </tr>
+    {% endif %}
+
+    {% include "bands/bandextra.html" %}
```

Comment:
Why did I have to include the bands/ URL even though this template is inside the bands folder? Who knows!! I certainly don't!